docs/reference/commandline/network_rm.md

```diff
@@ -10,18 +10,35 @@ parent = "smn_cli"
 
 # network rm
 
-    Usage:  docker network rm [OPTIONS] NAME | ID
+    Usage:  docker network rm [OPTIONS] NETWORK [NETWORK...]
 
-    Deletes a network
+    Deletes one or more networks
 
       --help=false       Print usage
 
-Removes a network by name or identifier. To remove a network, you must first disconnect any containers connected to it.
+Removes one or more networks by name or identifier. To remove the network, you must first disconnect any containers connected to it.
+
```

Comment:
Don't add a line here.  The example continues the thought of the para, it is fine.